Description: Declares the missing transitive dependencies required to run the tests
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
Index: libfreemarker-java/ivy.xml
===================================================================
--- libfreemarker-java.orig/ivy.xml
+++ libfreemarker-java/ivy.xml
@@ -96,6 +96,13 @@
       <!-- The lowerst supported xml-apis version depends on JDK version; prevent any collosion: -->
       <exclude org="xml-apis" module="xml-apis" />
     </dependency>
+
+    <dependency org="com.github.jnr" name="jnr-constants" rev="debian" conf="build.test->default; test->default" />
+    <dependency org="org.antlr" name="antlr" rev="3.2" conf="build.test->default; test->default" />
+    <dependency org="org.antlr" name="antlr-runtime" rev="3.2" conf="build.test->default; test->default" />
+    <dependency org="asm" name="asm-all" rev="3.3" conf="build.test->default; test->default" />
+    <dependency org="javax.servlet.jsp" name="javax.servlet.jsp-api" rev="2.3" conf="build.test->default; test->default" />
+    <dependency org="javax.servlet" name="javax.servlet-api" rev="3.1" conf="build.test->default; test->default" />
     
     <dependency org="javax.servlet.jsp" name="jsp-api" rev="1.2" conf="build.jsp1.2->default" />
     <dependency org="javax.servlet" name="servlet-api" rev="2.3" conf="build.jsp1.2->default" />
@@ -166,4 +173,4 @@
     <conflict org="javax.servlet.jsp" module="jsp-api" manager="strict" />
     
   </dependencies>
-</ivy-module>
\ No newline at end of file
+</ivy-module>
